src/lib/components/composites/input/InviteUsersInput.svelte

```diff
@@ -48,14 +48,13 @@
             .response.then(({ users }) => {
                 isErrorWhileLoadingCandidates = false;
 
-                const newInviteCandidates = users.filter(
-                    (user) => !isCandidateAlreadyInvitee(user.email),
-                );
-                const retainedCandidates = loadedInviteCandidates.filter((c) =>
-                    invitees.includes(c.email),
-                );
-
-                loadedInviteCandidates = [...newInviteCandidates, ...retainedCandidates];
+                const newInviteCandidates = users.filter((user) => !invitees.includes(user.email));
+
+                newInviteCandidates.forEach((candidate) => {
+                    if (!loadedInviteCandidates.includes(candidate)) {
+                        loadedInviteCandidates.push(candidate);
+                    }
+                });
```

Comment:
You kind of filter two times, while you could do this in one filter call here.
```suggestion
                const newInviteCandidates = users.filter((user) =>
                    !invitees.includes(user.email) && !loadedInviteCandidates.includes(user));
                loadedInviteCandidates.push(...newInviteCandidates);
```

Reply:
However, your suggestion has not the same behavior as I return the newInviteCandidates and in your version the user doesn't get a suggestion twice. So if you type "bob" then you get a suggestion and then type "bob I" and delete the "I" so have the input "bob" again. Then the suggestion for "bob" is not shown anymore, as it was already loaded and is filtered out. I already implemented your suggestion, but if I remember correctly I noticed the problem I described.

Reply:
I didn't find this problem. Then we should keep your solution.